README.md

```diff
@@ -123,62 +95,46 @@ gcloud auth application-default login
 
 ### Configure gcloud settings
 
+The scripts that manage the infrastructure and deployments for this demo attempt to initialize themselves with values from the `gcloud` config defaults for `compute/zone`, and `core/project`.
```

Comment:
Perhaps either force folks to do this config by calling a make target that verifies this is all set before calling make validate OR call it out more directly. IE: you cannot continue if these condition haven't been met. I lean towards automating and letting that code serve as forceful documentation.

Reply:
Ultimately the task was to add this existing project to Jenkins and I was trying not to expand the scope too much. I agree that it would be a nice improvement--I think the question is whether it needs to be part of this particular PR. Thoughts?